src/taskgraph/util/verify.py

```diff
@@ -223,6 +223,25 @@ def verify_routes_notification_filters(
                     )
                 )
 
+@verifications.add("full_task_graph")
+def verify_routes_invalid_slash(
+    task, taskgraph, scratch_pad, graph_config, parameters
+):
+    """
+    This function ensures that routes do not contain forward slashes.
+    """
+    if task is None:
+        return
+    task_dict = task.task
+    routes = task_dict.get("routes", [])
+
+    for route in routes:
+        # Check for invalid / in the route
+        if "/" in route:
```

Comment:
Sorry for the late request.. but it just occurred to me that only *index* routes can't have a slash. Routes in general are allowed to have a slash.

So we should change this to `if route.startswith("index.") and "/" in route:`. Also updating the function name to `verify_index_routes..` or similar.